### PR TITLE
added event emitter source in the log

### PR DESCRIPTION
**Pull request by carroaung**
added event emitter source in the log

### CODE REVIEW

File: config/event-emitter.php [hunk 1]
```diff
@@ -1,6 +1,11 @@
 <?php
 
 return [
+    /**
+     * Event emitter source name to be used when logging
+     */
+    'source' => env('EVENT_EMITTER_ENABLE', 'undefined'),
```
Comment (datpmwork) [inline config/event-emitter.php:7]: @carroaung we can use the `APP_NAME` env for default value if somewhere we forget to add it
And I think the name should be `EVENT_EMITTER_SOURCE`

```suggestion
    'source' => env('EVENT_EMITTER_SOURCE', env('APP_NAME', 'undefined')),
```

File: src/Jobs/Concerns/HasEmitterConcern.php [hunk 2]
```diff
@@ -13,10 +34,11 @@ trait HasEmitterConcern
     public function log($message, $context)
     {
         if (config('event-emitter.logging', false)) {
+            $source = $this->getSource() ?? 'undefined';
             $context = array_merge($context, [
                 'authUser' => data_get($this->getAuthUser(), 'id'),
             ]);
-            logger()->info("[Event Emitter] {$message}", $context);
+            logger()->info("[Event Emitter] Source $source {$message}", $context);
```
Comment (datpmwork) [inline src/Jobs/Concerns/HasEmitterConcern.php:41]: @carroaung should we add the source as one items of log context, like the authUser? 🤔

Comment (carroaung) [reply src/Jobs/Concerns/HasEmitterConcern.php]: https://github.com/carro-public/event-emitter/pull/4/commits/5cd96e18d3ef4834798e9f758100621a7a935cab

File: config/event-emitter.php (hunk 1 above)
Comment (carroaung) [reply config/event-emitter.php]: updated 

https://github.com/carro-public/event-emitter/pull/4/commits/5cd96e18d3ef4834798e9f758100621a7a935cab

**Review by datpmwork — APPROVED** [review]
LGTM

**Pull request merged**